package.json

```diff
@@ -66,8 +66,8 @@
   "devDependencies": {
     "after": "0.8.2",
     "connect-redis": "3.4.2",
-    "cookie-parser": "1.4.6",
-    "cookie-session": "2.0.0",
+    "cookie-parser": "1.4.7",
+    "cookie-session": "2.1.0",
```

Comment:
For all dep updates we are moving to `^` prefixes. This will need to be updated as well, but we prefer doing this in individual commits, so it is unlikely this PR would merge. You may want to open these independently so they can be merged.